src/common/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ add_library(lpf_common_${LPFLIB_CONFIG_NAME} OBJECT
 )
 
 
-add_gtest(time_test time.t.cpp time.cpp stack.cpp)
-add_gtest(memreg_test memreg.t.cpp log.cpp time.cpp config.cpp stack.cpp)
-add_gtest(sparseset_test sparseset.t.cpp log.cpp time.cpp config.cpp stack.cpp)
+add_gtest(time_test "pthread" OFF time.t.cpp time.cpp stack.cpp)
```

Comment:
note to self: "pthread" means only link against pthread engine, and OFF means do not link against lpf_debug